test/acceptance/tests/connect/connect_inject_test.go

```diff
@@ -11,54 +13,28 @@ import (
 
 const staticClientName = "static-client"
```

Comment:
Can we create a const for `staticServerName` as well and use that across the tests?

Reply:
yes, definitely 😄